metaflow/plugins/argo/argo_workflows.py

```diff
@@ -513,6 +530,7 @@ def _compile_workflow_template(self):
                     Metadata()
                     .label("app.kubernetes.io/name", "metaflow-run")
                     .label("app.kubernetes.io/part-of", "metaflow")
+                    .label("workflows.argoproj.io/workflow-template", self.name)
```

Comment:
custom label name to avoid clashes with argo

Reply:
Done. changed to custom label and using hash of name as a value.